src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Logging/SerilogWrapper.cs

```diff
@@ -37,6 +38,26 @@ public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
 
         public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
         {
+            var frame7 = new System.Diagnostics.StackFrame(7, false).GetMethod().Name;
```

Comment:
I think a comment here explaining the issue would be great

Reply:
Yes, a comment would be helpful.